packages/front-end/components/Experiment/ResultsTable.tsx

```diff
@@ -133,14 +133,14 @@ export default function ResultsTable({
               <th
                 className={clsx(
                   "value pt-2",
-                  `variation${i} head-last-row text-center`
+                  `variation${v.id} head-last-row text-center`
```

Comment:
id is likely unsafe here

Reply:
for posterity, we decided to change this in your branch